.github/CODEOWNERS

```diff
@@ -71,22 +105,52 @@
 /docs/framework/configure-apps/file-schema/wcf/** @HongGit
 /docs/framework/data/wcf/** @HongGit
 /docs/framework/wcf/** @HongGit
-# WPF
-/docs/framework/wpf/** @Thraka
-# Winforms
-/docs/framework/winforms/** @Thraka
 
 ################## .NET STANDARD ##################
-# Analyzers:
-/docs/standard/analyzers/** @BillWagner
+# Analyzers
+/docs/standard/analyzers/** @mairaw
+# Assembly
+/docs/standard/assembly/** @gewarren
+# Asynchronous Programming Patterns
+/docs/standard/asynchronous-programming-patterns/** @gewarren
+# Attributes
+/docs/standard/attributes/** @tdykstra
+# Base types
+/docs/standard/base-types/** @Thraka
+# Collections
+/docs/standard/base-types/** @BillWagner
+# Base types
+/docs/standard/base-types/** @Thraka
+# Data XML
+/docs/standard/data-xml/** @gewarren
+# Datetime
+/docs/standard/datetime/** @Thraka
+# Events
+/docs/standard/exceptions/** @gewarren
+# Exceptions
+/docs/standard/exceptions/** @tdykstra
+# GC
+/docs/standard/garbage-collection/** @BillWagner
+# Generics
+/docs/standard/generics/** @Thraka
+# Globalization
+/docs/standard/globalization-localization/** @mairaw
+# I/O
+/docs/standard/io/** @Thraka
+# LINQ
+/docs/standard/linq/** @mairaw
```

Comment:
I don't see a linq folder under standard.

Reply:
It's coming with a vendor task.